lib/tasks/code_analysis.rake

```diff
@@ -1,6 +1,6 @@
-task :code_analysis do
+task code_analysis: :environment do
   sh 'bundle exec brakeman . -z -q'
-  sh 'bundle exec rubocop app config lib spec'
-  sh 'bundle exec reek app config lib'
+  sh 'bundle exec rubocop .'
+  sh 'bundle exec reek app lib public spec tmp'
```

Comment:
why not `bundle exec reek .` ?

Reply:
that actually broke circleCI. If reek used the `.` scope, it actually started analyzing the gems. Here you can see an example run: https://app.circleci.com/pipelines/github/rootstrap/rails_api_base/75/workflows/ddb79150-954d-46d3-9364-edb7baa6d2ea/jobs/747

Reply:
when running `bundle exec reek .` in the local env, there are no problems. The problems occur only in circleCI. Maybe this should be addressed before merging this PR by changing `.circleci/config.yml`. What do you think?

Reply:
The issue there is that reek was run against the bundled dependencies, I guess it's happening the same with rubocop. 

So we should exclude the path where our gems are bundled (in circleCI)